cmd/multiFaToVcf/multiFaToVcf.go

```diff
@@ -20,14 +20,20 @@ func multiFaToVcf(inFile string, chr string, outFile string, substitutionsOnly b
 	out := fileio.EasyCreate(outFile)
 	header := vcf.NewHeader()
 	vcf.NewWriteHeader(out, header)
-	convert.PairwiseFaToVcf(f, chr, out, substitutionsOnly, retainN)
+	if len(f) == 2 {
+		convert.PairwiseFaToVcf(f, chr, out, substitutionsOnly, retainN)
+	} else if len(f) == 3 {
+		convert.ThreeWayFaToVcf(f, chr, out, substitutionsOnly, retainN)
+	}
```

Comment:
we might need an else case here if it's not 2 or 3, and the else case should lead to a log.Fatalf